package.json

```diff
@@ -481,102 +462,92 @@
                     "group": "regs_repo_1_general@1"
                 },
                 {
                     "command": "vscode-docker.registries.azure.deleteRepository",
-                    "when": "view == dockerRegistries && viewItem == azure;DockerV2;Repository;",
+                    "when": "view == dockerRegistries && viewItem =~ /azureContainerRepository/i",
                     "group": "regs_repo_2_destructive@1"
                 },
                 {
                     "command": "vscode-docker.registries.pullImage",
-                    "when": "view == dockerRegistries && viewItem =~ /Tag;/",
+                    "when": "view == dockerRegistries && viewItem =~ /commontag/i",
                     "group": "regs_tag_1_general@1"
                 },
                 {
                     "command": "vscode-docker.registries.copyRemoteFullTag",
-                    "when": "view == dockerRegistries && viewItem =~ /(DockerV2|DockerHubV2);Tag;/",
+                    "when": "view == dockerRegistries && viewItem =~ /commontag/i",
                     "group": "regs_tag_1_general@2"
                 },
                 {
                     "command": "vscode-docker.registries.copyImageDigest",
-                    "when": "view == dockerRegistries && viewItem =~ /DockerV2;Tag;/",
+                    "when": "view == dockerRegistries && viewItem =~ /registryV2Tag/i",
                     "group": "regs_tag_1_general@3"
                 },
                 {
                     "command": "vscode-docker.registries.deployImageToAzure",
-                    "when": "view == dockerRegistries && viewItem =~ /(DockerV2|DockerHubV2);Tag;/ && isAzureAccountInstalled",
+                    "when": "view == dockerRegistries && viewItem =~ /commontag/i",
                     "group": "regs_tag_1_general@4"
                 },
                 {
                     "command": "vscode-docker.registries.deployImageToAca",
-                    "when": "view == dockerRegistries && viewItem =~ /(DockerV2|DockerHubV2|GitLabV4);Tag;/ && isAzureAccountInstalled",
+                    "when": "view == dockerRegistries && viewItem =~ /commontag/i",
                     "group": "regs_tag_1_general@6"
                 },
                 {
                     "command": "vscode-docker.registries.azure.untagImage",
-                    "when": "view == dockerRegistries && viewItem == azure;DockerV2;Tag;",
+                    "when": "view == dockerRegistries && viewItem =~ /azureContainerTag/i",
                     "group": "regs_tag_2_destructive@1"
                 },
                 {
                     "command": "vscode-docker.registries.deleteImage",
-                    "when": "view == dockerRegistries && viewItem =~ /DockerV2;Tag;/",
+                    "when": "view == dockerRegistries && viewItem =~ /(genericRegistryV2Tag|azureContainerTag)/i",
```

Comment:
Should this be `registryV2Tag` instead? Or maybe even `commontag`? It's possible for any provider to support, though not all of them will (in which case, it's fine if we throw an error)

Reply:
yeah that's a great point. i know we currently support it in all `registryV2Tag`. I'm a little worries about the UX for gitlab, dockerhub, and github. Could we do something like:

`viewItem =~ /commontag/ i && !(viewItem =~ /(gitLabTag|gitHubTag|dockerHubTag)/i)`

Reply:
I think that's reasonable